site2/docs/concepts-multiple-advertised-listeners.md

```diff
@@ -23,9 +23,8 @@ This example shows how a Pulsar client uses multiple advertised listeners.
 
 1. Configure multiple advertised listeners in the broker configuration file.
 
-```shell
-advertisedListeners={listenerName}:pulsar://xxxx:6650,
-{listenerName}:pulsar+ssl://xxxx:6651
+```conf
```

Comment:
Did we plan to change `conf` to `properties` gradually?

Reply:
`properties` seems to be more accurate, I'll change it.